Yoast/wordpress-seo
# Redesign analysis

## Summary

This PR can be summarized in the following changelog entry:

## Relevant technical choices:

* Use JS for moving the helpbutton to the metabox section where it belongs to.

## Test instructions

This PR can be tested by following these steps:

* Open the metabox and for the posts and check if the styling has been applied. Do thisfor the taxonomy also. Be sure everything still works.
